src/Microsoft.AspNet.Mvc.Core/OptionDescriptors/ModelBinderDescriptorExtensions.cs

```diff
@@ -80,5 +80,24 @@ public static class ModelBinderDescriptorExtensions
             descriptors.Insert(index, descriptor);
             return descriptor;
         }
+
+
+        /// <summary>
+        /// Removes all instances of <typeparamref name="TInstance"/> from a list of ModelBinderDescriptors.
+        /// </summary>
+        /// <typeparam name="TInstance">A type that represents an <see cref="IModelBinder"/>.</typeparam>
```

Comment:
nit: tend to avoid "represents".  so perhaps `The <see cref="ModelBinderDescriptor"/> sub-type of instances to remove.`  not positive `<see cref="ModelBinderDescriptor"/> sub-` is that helpful.  up to you.